internal/apps/devflow/flow/flow.service_test.go

```diff
@@ -711,3 +713,63 @@ func Test_canJoin(t *testing.T) {
 		})
 	}
 }
+
+type issueForCreateEventMock struct {
+	IssueMock
+}
+
+func (i issueForCreateEventMock) GetIssue(id int64, identityInfo *commonpb.IdentityInfo) (*issuepb.Issue, error) {
+	if id == 0 {
+		return nil, fmt.Errorf("error")
```

Comment:
should point out concrete error even in mock

Reply:
done